components/kyma-environment-broker/internal/process/upgrade_kyma/secret_cm_overrides.go

```diff
@@ -50,7 +50,7 @@ func (s *OverridesFromSecretsAndConfigStep) Run(operation internal.UpgradeKymaOp
 
 	version, err := s.getRuntimeVersion(operation)
 	if err != nil {
-		return s.operationManager.RetryOperation(operation, "error while getting runtime version", err, 5*time.Second, 5*time.Minute, log)
+		return s.operationManager.OperationFailed(operation, "error while getting runtime version", err, log)
```

Comment:
Isn't it reflected in any test?

Reply:
The tests are covering only successfully executions.